src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,90 @@
+import React, {useState, useEffect} from 'react';
+import './Pagination.css'
+
+interface PaginationProps {
+  data: any[];
+  RenderComponent: React.FC<any>;
+  title?: string;
+  pageLimit: number;
+  dataLimit: number;
+}
+
+
+function Pagination ({data, RenderComponent, title, pageLimit, dataLimit}: PaginationProps) {
+  const [pages] = useState(Math.round(data.length / dataLimit));
+  const [currentPage, setCurrentPage] = useState(1);
```

Comment:
This is fine for now, but when we start the refactoring stage we'll have to move all of our state to the reducer, just something to keep in mind.